.gitignore

```diff
@@ -3,4 +3,10 @@ workspace
 .metadata/*
 .project
 .recommenders/*
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
```

Comment:
I am fine to add the pattern, but I think it is enough to add the pattern _once_ at the root level. It will ignore any files in the sub-directories (just as the ```.DS_Store``` matches any such file in any subdirectory)

Reply:
Thank you. That is simple and elegant. I have committed and push the code.